examples/twittermap/web/app/controllers/TwitterMapApplication.scala

```diff
@@ -91,8 +100,90 @@ class TwitterMapApplication @Inject()(val wsClient: WSClient,
     }
   }
 
+
+  object LiveTweetActor {
```

Comment:
We decided to use `Search` API instead of `Streaming` API to retrieve live tweets in case multiple concurrent clients may block one Twitter Developer Account.